Documentation/api-reference.md

```diff
@@ -70,6 +70,9 @@ Edge.getLocationHint(new AdobeCallbackWithError<String>() {
 
 Registers the Edge Network extension with the Mobile Core extension.
 
+> **Warning**  
+> Deprecated as of 2.0.0. Use the [MobileCore.registerExtensions API](https://github.com/adobe/aepsdk-core-android) instead.
```

Comment:
update url: https://github.com/adobe/aepsdk-core-android/blob/staging/Documentation/MobileCore/api-reference.md

Reply:
let's use main branch for all link references:
https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/api-reference.md